modules/nextflow/src/main/groovy/nextflow/executor/WrExecutor.groovy

```diff
@@ -0,0 +1,664 @@
+/*
+ * Copyright 2019, Genome Research Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.executor
+
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+import java.nio.file.Path
+import java.nio.file.Paths
+import javax.net.ssl.HostnameVerifier
+import javax.net.ssl.HttpsURLConnection
+import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManager
+import javax.net.ssl.X509TrustManager
+import nextflow.exception.AbortOperationException
+import nextflow.processor.TaskHandler
+import nextflow.processor.TaskMonitor
+import nextflow.processor.WrMonitor
+import nextflow.processor.TaskRun
+import nextflow.processor.TaskBean
+import nextflow.processor.TaskStatus
+import nextflow.processor.BatchContext
+import nextflow.processor.BatchHandler
+import nextflow.util.Escape
+import groovy.json.JsonSlurper
+import static groovy.json.JsonOutput.toJson
+/**
+ * Executor that schedules jobs using wr as a backend, avoiding storage of
+ * state on disk.
+ *
+ * See https://github.com/VertebrateResequencing/wr
+ *
+ * @author Sendu Bala <sb10@sanger.ac.uk>
+ * Based on TesExecutor by Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ */
+@Slf4j
+@CompileStatic
+class WrExecutor extends Executor {
+
+	static private String token
+	static private String defaultManagerDir
+    static private String endpoint
+	static private String cacertPath
+	static private WrRestApi client
+
```

Comment:
It looks there's a problem with tab/space indentation. Please use 4 spaces in place of tabs. More details [here](https://github.com/nextflow-io/nextflow/blob/master/CONTRIBUTING.md).

Reply:
Should now be fixed